engine/core/pom.xml

```diff
@@ -81,6 +81,17 @@
 			<artifactId>asm5</artifactId>
 		</dependency>
 
+        <dependency>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-jsonSchema</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.eobjects.datacleaner</groupId>
+            <artifactId>DataCleaner-rest-client</artifactId>
```

Comment:
Shouldn't the dependencies be inverted here? Now "engine-core" is depending on "rest-client". I would expect that it was opposite: That "rest-client" depended on "engine-core" and that the Transformer implementations etc. are created in that module?